packages/client3/components/design-system/input/input.tsx

```diff
@@ -112,22 +106,15 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
       <div className="relative w-full">
         <input
           className={clsx(
-            "unfocused w-full rounded", // unfocused because the color schemes supply a border color as a focus style
-            colorScheme === "light"
-              ? [
-                  "border bg-white focus:border-sand-600",
-                  isError
-                    ? "border-clay-100 placeholder:text-clay-700"
-                    : "border-sand-200 placeholder:text-sand-500",
-                ]
-              : colorScheme === "dark"
-              ? [
-                  "border bg-sky-900 focus:border-white",
-                  isError
-                    ? "border-clay-500 placeholder:text-clay-500"
-                    : "border-transparent placeholder:text-sand-300",
-                ]
-              : null,
+            "unfocused w-full rounded border",
+            // unfocused because the color schemes supply a border color as a focus style
```

Comment:
Now that color scheme is removed from props, does this comment hold value?

Reply:
Fair enough. Taken out.